src/main/java/net/reikeb/notenoughgamerules/Gamerules.java

```diff
@@ -42,6 +45,9 @@ public static void setupGamerules() {
         ANVIL_DAMAGE = GameRuleRegistry.register("anvilDamage", GameRules.Category.PLAYER, GameRuleFactory.createBooleanRule(true));
         CAN_COPPER_OXIDE = GameRuleRegistry.register("canCopperOxide", GameRules.Category.UPDATES, GameRuleFactory.createBooleanRule(true));
         CAN_HURT_PET_MOBS = GameRuleRegistry.register("canHurtPetMobs", GameRules.Category.MOBS, GameRuleFactory.createBooleanRule(true));
+        CAN_PLAYER_INTERACT_WITH_BLOCK = GameRuleRegistry.register("canPlayerInteractWithAction", GameRules.Category.PLAYER, GameRuleFactory.createBooleanRule(true));
```

Comment:
Please rename the gamerule ID to `canPlayerInteractWithBlock`

Reply:
oh i forgot that